kitsune/wiki/models.py

```diff
@@ -332,14 +332,20 @@ def __setattr__(self, name, value):
         """Trap setting slug and title, recording initial value."""
         # Public API: delete the old_title or old_slug attrs after changing
         # title or slug (respectively) to suppress redirect generation.
-        if getattr(self, 'id', None):
```

Comment:
Any specific reason to remove it?

Reply:
I made this change to avoid a recursive `__setattr__` situation that showed up during testing. The following line testing `self._state.adding` has the same logical effect.